sotodlib/core/axisman.py

```diff
@@ -835,6 +835,7 @@ def save(self, dest, group=None, overwrite=False):
             dest).
           overwrite (bool): If True, remove any existing thing at the
             specified address before writing there.
+          compression (str ot None): Compression filter to apply.
```

Comment:
str _or_ None.

Add "E.g. 'gzip'. This string is passed directly to HDF5 dataset routines."